### DISCUSSION TITLE
When using --service crunchy and --dubLang eng jpn only the first language is being honored?

### DISCUSSION THREAD

**Discussion by RickeyWard** [anidl/multi-downloader-nx#791] [Q&A] [answered]
I can download jpn or eng using --dubLang but it doesn't download both? if I use the GUI I get both languages as expected but I can't seem to pass --syncTiming and all the english audio is off.

What am I doing wrong?
(there was a dead discussion that is locked with no answer)

**Comment by Jaynator495** [comment]
Are you using --srz or -s?

**Reply by RickeyWard** [reply]
-s yes, I'm using it like

```
./AniDL --service crunchy -s GYDQ27Z36 -e 1 --syncTiming --chapters --dubLang jpn eng
```

**Reply by Jaynator495** [reply, marked as answer]
Gotcha, -s currently doesn't support multi dub for crunchy, use --srz

**Reply by RickeyWard** [reply]
Thanks for the answer!
